Auto3/Auto3.ino

```diff
@@ -1351,57 +1386,158 @@ void eepromUpdate()
   }
 }
 
-int firstValue(){
+void errorReport(byte errorType, int refID)
```

Comment:
What is the purpose of this function?

Reply:
Function gives a all in one error reporting function for centralization.